docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
         (
           cd grapl
           pulumi stack init local-grapl --non-interactive
-          pulumi up --yes --skip-preview --stack=local-grapl $$TRACING_ARGS
```

Comment:
I've removed the skip preview since its no longer necessary and it acts as a check for new pulumi preview breakage